configfiles/fit/config_Lc_Fit_pp13TeV.yml

```diff
@@ -0,0 +1,20 @@
+pp5TeVFD:
+    #Meson: 'Lc'
+    FixSigma: 1
+    SigmaFile: '../../Analyses/pp5TeV/Ds_wML_mult/outputs/100320/raw_yield/RawYield_Ds_data_norm_pt2_12.root'
+    SigmaMultFactor: 1. #float or MinusUnc, PlusUnc
+    FixMean: 0
+    MeanFile: ""
+    PtMin: [4, 6]
+    PtMax: [6, 12]
+    MassMin: [1.78, 1.78]
+    MassMax: [2.10, 2.10]
+    Rebin: [6, 8]
+    InclSecPeak: [1, 1, 1, 1]
```

Comment:
this should be all 0 for each pT bin for the Lc (it should be 1 only for the Ds)